src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+
+  const pricePerDay = 40;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+  let cost = days * pricePerDay;
+
+  if (days >= 3 && days < 7) {
+    cost = cost - smallDiscount;
+  }
+
+  if (days >= 7) {
+    cost = cost - bigDiscount;
+  }
```

Comment:
Этот блок можно упростить
```suggestion
  if (days < 3) {
    return cost;
  }

  if (days <= 7) {
    cost = cost - smallDiscount;
  }
```
Если ни один из ифов не сработал, значит дней больше 7, и нужно вернуть cost - bigDiscount